index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header>
+            <a class="logo" href="#"><p>My logo</p></a>
```

Comment:
- [ optional] A recommend using your own logo or name if you don't have a logo since this will be you own portfolio where you will showcase all your projects.